os/dbuild.sh

```diff
@@ -109,6 +109,9 @@ function SELECT_OPTION()
 			echo "  \"4. Clean Build\""
 			echo "  \"5. Clean Build and Re-Configure\""
 			echo "  \"6. Build SmartFS Image\""
+			if [ "${STATUS}" == "BUILT" ]; then
+			echo "  \"7. Run TRAP tool\""
```

Comment:
How about using something like "**Run crash log parser**" instead of TRAP? I feel like it will be more user-friendly.